public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ResourcePicker/ResourcePicker.tsx

```diff
@@ -216,44 +229,7 @@ const ResourcePicker = ({
           </>
         )}
 
-        <Collapse
-          collapsible
-          label="Advanced"
-          isOpen={isAdvancedOpen}
-          onToggle={() => setIsAdvancedOpen(!isAdvancedOpen)}
-        >
-          <Label htmlFor={`input-${internalSelectedURI}`}>
-            <h6>
-              Resource URI{' '}
-              <Tooltip
-                content={
-                  <>
-                    Manually edit the{' '}
-                    <a
-                      href="https://docs.microsoft.com/en-us/azure/azure-monitor/logs/log-standard-columns#_resourceid"
-                      rel="noopener noreferrer"
-                      target="_blank"
-                    >
-                      resource uri.{' '}
-                    </a>
-                    Supports the use of multiple template variables (ex: /subscriptions/$subId/resourceGroups/$rg)
-                  </>
-                }
-                placement="right"
-                interactive={true}
-              >
-                <Icon name="info-circle" />
-              </Tooltip>
-            </h6>
-          </Label>
-          <Input
-            id={`input-${internalSelectedURI}`}
-            value={internalSelectedURI}
-            onChange={(event) => setInternalSelectedURI(event.currentTarget.value)}
-            placeholder="ex: /subscriptions/$subId"
-          />
-          <Space v={2} />
-        </Collapse>
+        <Advanced resource={internalSelected} onChange={(r) => setInternalSelected(r)} />
```

Comment:
Should we be more specific on the naming?

Maybe `AdvancedResourcePicker` or `AdvancedResourceSelector` ? Or is the rational that because it lives under `ResourcePicker` folder, that is implicit?

Reply:
I think so, at least other components here follow the same approach e.g.  `NestedRow` is not `ResourcePickerNestedRow`, it's implicit (and would be a bit tedious if not).